.git-blame-ignore-revs

```diff
@@ -10,4 +11,5 @@
 # Note that these features require Git 2.23 (released August 2019).
 03cabb7f55d54e2b09eefd0e03df2d440f962bc4
 bc96c774be6e307e8e3e4d39780d37b045d0973a
+2249eebd81b195c72c9293a1a2224fa1134d4ca3
```

Comment:
The idea here is that we didn't want to misattribute the authorship of lines when running the *initial* big syntax-cleanup pass; I don't think that you need to add this here; you actually authored all the lines you're reformatting.  (It's fine if you think it's helpful for blame legibility, this PR is big and quite old, I just really don't want to establish the precedent that everyone who runs `black` starts appending to this file.)